.travis.yml

```diff
@@ -33,7 +33,7 @@ deploy:
   provider: script
   script: heroku container:release web --app $HEROKU_APP
   on:
-    branch: Module-9
+    branch: Module-10
```

Comment:
(Don't forget to switch back to main when merging)

Reply:
Yes I have now amended it to master.